### PR TITLE
Rename 'mission' to 'theme'

### PR DESCRIPTION
**Pull request by jon-kirwan**
## What

Rename the `MissionTypeHelper`, `mission_type` property, and `mission-` Sass variable(s) to theme

## Why

To make them more generic

### CODE REVIEW

**Review by leenagupte — COMMENTED** [review]
Just a couple of inline comments. Things that were missed previously. Otherwise it looks good.

**Comment by matthillco** [comment]
Frontend code looks fine. 👍🏻

File: app/views/landing_page/blocks/_card.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <%
   add_view_stylesheet("landing_page/card")
 %>
-<div class="app-b-card <%= "border-top--#{MissionTypeHelper.style(block.data["mission_type"])}" %>">
+<div class="app-b-card <%= "border-top--#{ThemeTypeHelper.style(block.data["theme_type"])}" %>">
```
[the remark this answers is not in the document]
Comment (matthillco) [reply app/views/landing_page/blocks/_card.html.erb:4]: Also, do we even need `-type`? `theme` seems fine by itself.

File: app/views/landing_page/blocks/_hero.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
   grid_class = "govuk-grid-column-one-third-from-desktop" if block.data["theme"] == "middle_left"
 
   hero_textbox_classes = %w[app-b-hero__textbox]
-  hero_textbox_classes << "border-top--#{MissionTypeHelper.style(4)}" if block.data["mission_type"]
+  hero_textbox_classes << "border-top--#{ThemeTypeHelper.style(4)}" if block.data["theme_type"]
```
[the remark this answers is not in the document]
Comment (matthillco) [reply app/views/landing_page/blocks/_hero.html.erb:12]: As above, do we even need `-type`? `theme` seems fine by itself.

**Review by matthillco — COMMENTED** [review]
A couple of comments about `theme_type`. I think `theme` alone would be fine, which is a bit neater overall.

**Comment by jon-kirwan** [comment]
> A couple of comments about `theme_type`. I think `theme` alone would be fine, which is a bit neater overall.

Thanks Matt. Unfortunately that might cause some problems though as I now realise we've got a `theme` property name for the grid column size? Maybe something like `theme_colour`?

**Review by matthillco — APPROVED** [review]
Looks good, nice one.

**Pull request merged**